docs/source/json-api/index.rst

```diff
@@ -1234,6 +1234,424 @@ HTTP Response
+      "rights": [
+        {
+          "type": "CanActAs",
+          "party": "Carol"
+        },
+        {
+          "type": "CanReadAs",
+          "party": "Alice",
+        },
+        {
+          "type": "CanReadAs",
+          "party": "Bob",
+        },
+        {
+          "type": "ParticipantAdmin"
+        }
+      ]
+    }
+
+Please refer to :ref:`CreateUserRequest <com.daml.ledger.api.v1.admin.user_management_service.CreateUserRequest>` documentation for information about the meaning of the fields.
+
+All fields in the request are required, this means that an empty JSON object is not a valid request to allocate a new user.
+
+HTTP Response
+=============
+
+.. code-block:: json
+
+    {
+      "result": true,
+      "status": 200
+    }
+
+
+Get Authenticated User Information
+**********************************
+
+This endpoint is a JSON API proxy for the Ledger API's :ref:`GetUserRequest <com.daml.ledger.api.v1.admin.user_management_service.GetUserRequest>`. For more information about user management, please refer to :ref:`Provisioning Identifiers <provisioning-ledger-identifiers>` part of the Ledger API documentation.
+
+The user id will always be filled out with the user specified via the currently used User Token.
+
+HTTP Request
+============
+
+- URL: ``/v1/user``
+- Method: ``GET``
+
+HTTP Response
+=============
+
+.. code-block:: json
+
+    {
+      "result": {
+        "userId": "Carol",
+        "primaryParty": "Carol",
+      },
+      "status": 200
+    }
+
+
+Get Specific User Information
+*****************************
+
+This endpoint is a JSON API proxy for the Ledger API's :ref:`GetUserRequest <com.daml.ledger.api.v1.admin.user_management_service.GetUserRequest>`. For more information about user management, please refer to :ref:`Provisioning Identifiers <provisioning-ledger-identifiers>` part of the Ledger API documentation.
+
+HTTP Request
+============
+
+- URL: ``/v1/user``
+- Method: ``POST``
+- Content-Type: ``application/json``
+- Content:
+
+.. code-block:: json
+
+    {
+      "userId": "Carol"
+    }
+
+
+Please refer to :ref:`GetUserRequest <com.daml.ledger.api.v1.admin.user_management_service.GetUserRequest>` documentation for information about the meaning of the fields.
+
+HTTP Response
+=============
+
+.. code-block:: json
+
+    {
+      "result": {
+        "userId": "Carol",
+        "primaryParty": "Carol",
+      },
+      "status": 200
+    }
+
+Delete Current User
+*******************
+
+This endpoint is a JSON API proxy for the Ledger API's :ref:`DeleteUserRequest <com.daml.ledger.api.v1.admin.user_management_service.DeleteUserRequest>`. For more information about user management, please refer to :ref:`Provisioning Identifiers <provisioning-ledger-identifiers>` part of the Ledger API documentation.
+
+The user id will always be filled out with the user specified via the currently used User Token.
+
+HTTP Request
+============
+
+- URL: ``/v1/user/delete``
+- Method: ``GET``
```

Comment:
Even though our API is not RESTful, in general I believe `GET` request are expected to be non-destructive. Is there a reason why we decided not to make a `POST` out of this? /cc @S11001001

Reply:
This should indeed be a POST.

Reply:
I think I'd prefer not having this at all. Having to supply a body seems less error prone than having a different behaviour based on whether you supply a body or not (i.e. only keep the "Delete Specific User" one).

Reply:
We decided to remove that GET endpoint, therefor will remove the documentation here.